src/ray/gcs/gcs_client/gcs_client.cc

```diff
@@ -199,6 +197,11 @@ Status PythonGcsClient::Connect(const ClusterID &cluster_id,
   }
 
   RAY_CHECK(!cluster_id_.IsNil()) << "Unexpected nil cluster ID.";
+  kv_stub_ = rpc::InternalKVGcsService::NewStub(channel_);
```

Comment:
why do we not reset things like `node_info_stub_`?

Reply:
We initialize `node_info_stub_` at the time of creating the channel, and reset it within the loop, so by this point it should already be initialized. The remaining stubs here have not been initialized, so we do that here.